src/DB/projects.json

```diff
@@ -1,4 +1,23 @@
 [
+  {
+      "github_username": "PatilHarshh",  
+      "Social_media": {
+        "gitHub": "https://github.com/PatilHarshh",
+        "LinkedIn": "https://www.linkedin.com/in/harsh-patil-1005061b4/",
+        "Instagram": "YOUR_INSTAGRAM_ACCOUNT_LINK",
+        "YouTube": "YOUR_YOUTUBE_ACCOUNT_LINK",
+        "Twitter": "https://twitter.com/Harsh94569157"
+      },
+      "Projects": [
+        {
+          "link": "https://github.com/PatilHarshh/Unit-Converter",
+          "title": "Unit-Converter",
+          "description": "This is the Unit Converter Project Basic Logic of Javascript and Begineer Friendly Project",
+          "tech": ["Javascript", "HTML","CSS"]
+        }
```

Comment:
you already added your project before so just paste the code to the previous one. check the contributing guide

Reply:
ok